### DISCUSSION TITLE
--check-files equivalent

### DISCUSSION THREAD

**Discussion by RedTn** [yarnpkg/berry#4427] [Q&A] [answered]
Hi,

Is there a --check-files equivalent of https://classic.yarnpkg.com/lang/en/docs/cli/install/#toc-yarn-install-check-files in berry?

**Comment by arcanis** [comment, marked as answer]
No, `--check-files` is now irrelevant as Yarn always generate a consistent node_modules.

**Reply by RedTn** [reply]
Ok, on a side note then if I ever made manual changes to node_modules or some side effect caused it to change, is there a way to restore it other than deleting node_modules and running `yarn install` again?
